packages/bluebird/src/bootstrap-sass/overrides/_input-groups.scss

```diff
@@ -1,21 +1,202 @@
-.input-group-addon {
-  padding: 0 $xs 0 0;
-  font-size: $font-size-sm;
-  font-weight: $font-weight-normal;
-  line-height: 1;
-  color: $gray-base;
-  text-align: center;
-  background-color: transparent;
-  border: 0;
-  border-bottom: $standard-border;
-  border-radius: 0;
-
-  .icon {
-    position: relative;
-    top: 2px;
+// //
```

Comment:
There's a lot of commented out code in this file. Is there a reason to leave it in rather than remove it?

Reply:
https://github.com/enduranceinternational/bluebird/pull/16#discussion_r328812034

Reply:
https://github.com/enduranceinternational/bluebird/pull/16#discussion_r328812034